test/e2e/alertmanager_test.go

```diff
@@ -203,6 +206,15 @@ func TestAlertmanagerKubeRbacProxy(t *testing.T) {
 	if !ok {
 		t.Fatalf("couldn't get silenceID from response %q", string(b))
 	}
+	t.Cleanup(func() {
+	   resp, err := clients["editor"].Do("DELETE", fmt.Sprintf("/api/v2/silence/%s", silID), sil)
+	   if err != nil || resp.Status != 200 {
+	    t.Error("failed to delete silence")
+	   }
+	})
+		clients["editor"].Do("DELETE", fmt.Sprintf("/api/v2/silence/%s", silID), sil)
+		t.Log("deleting silence as 'editor' for cleanup")
+	}()
```

Comment:
think these three lines can be removed

Reply:
Certainly...directly applying a suggestion got me here I think.